src/unstract/adapters/vectordb/helper.py

```diff
@@ -91,13 +93,13 @@ def get_collection_name(
 
         """
         vector_db_collection_name: str = VectorDbConstants.DEFAULT_VECTOR_DB_NAME
-        if collection_name_prefix is not None and embedding_dimension is not None:
+        if embedding_dimension is not None:
```

Comment:
Can embedding_dimension be 0? If not this suggestion can be done.
```suggestion
        if embedding_dimension:
```